docs/usage/tables.md

```diff
@@ -12,19 +12,7 @@ all the tables.
 ## The metadata store
 Each table has metadata associated to it. This metadata is stored in a
 StateStore and consists of information about files that are in the 
-system, and the partitions. Sleeper has two options for the storage of
-this metadata: the `DynamoDBStateStore` and the `S3StateStore`. The default
-option is the `S3StateStore`.
-
-The `DynamoDBStateStore` stores the metadata in 3 DynamoDB tables. This
-allows quick updates and retrieval of the metadata. When a compaction
-job finishes, it needs to update the metadata about the files. Specifically
-it needs to mark the files that were input to the compaction job as being
-ready for garbage collection and it needs to add the output file as a new
-active file. This needs to happen in a transaction to ensure that clients
-always see a consistent view of the metadata. DynamoDB transactions are
-limited in size and this limits the number of files that can be read in a
-compaction to 11.
+system, and the partitions.SThe storage of is in `S3StateStore`. 
```

Comment:
This needs some work. It would be nice to explain the transaction log state store here if we can but we could make that a separate issue.

Reply:
I've raised this issue:
- https://github.com/gchq/sleeper/issues/4224